scripts/update_specs_from_pricing.py

```diff
@@ -234,28 +204,25 @@ def get_rds_pricing():
 
 
 def get_neptune_pricing():
-    results = {}
-    for page in get_paginator('AmazonNeptune'):
-        for price_item in page.get('PriceList', []):
-            products = json.loads(price_item)
-            product = products.get('product', {})
-            if product:
-                if product.get('productFamily') == 'Database Instance':
-                    if not results.get(region_map[product.get('attributes').get('location')]):
-                        results[region_map[product.get('attributes').get('location')]] = set()
-                    usage_type = product.get('attributes').get('usagetype').split(':')[1]
-                    results[region_map[product.get('attributes').get('location')]].add(usage_type)
```

Comment:
[switched from parsing `usage_type` to using `instanceType`](https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonNeptune/current/index.json)